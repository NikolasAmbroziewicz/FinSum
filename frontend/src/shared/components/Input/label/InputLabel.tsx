export interface IInputLabel {
  value: string;
}

const InputLabel: React.FC<IInputLabel> = ({ value }) => {
  return (
    <label
      className="block text-gray-600 text-sm font-bold mb-2"
      htmlFor={value}
    >
      {value}
    </label>
  );
};

export default InputLabel;
