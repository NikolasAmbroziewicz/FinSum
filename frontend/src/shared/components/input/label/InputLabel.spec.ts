import { describe, expect, it } from 'vitest';
import InputLabel from './InputLabel';
import { mockInputLabelProps } from './InputLabel.mock';

describe('InputLabel', () => {
  it('define', () => {
    expect(InputLabel(mockInputLabelProps.base)).toMatchSnapshot();
  });
});
