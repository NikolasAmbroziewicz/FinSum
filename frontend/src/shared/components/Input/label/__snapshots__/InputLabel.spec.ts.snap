// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`InputLabel > define 1`] = `
<label
  className="block text-gray-600 text-sm font-bold mb-2"
  htmlFor="Login"
>
  Login
</label>
`;
